### PR TITLE
Integrated the UI with backend and made some UI changes

### PR DESCRIPTION
**Pull request by Akshit-Accolite**
Integrated the UI with backend and made some UI changes

### CODE REVIEW

File: src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java [hunk 1]
```diff
@@ -187,8 +191,13 @@ public void actionPerformed(ActionEvent e) {
                 onPressedChangeVersion();
             }
         });
-
```
Comment (vaibhavjain3) [inline src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java:190]: Line break should be there.

File: src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java [hunk 2]
```diff
@@ -393,12 +412,55 @@ public JPanel initHelpUI() {
...
+            versionInputModel.setNewVersion(textNewVersionStringTS.getText());
+        if (textGUIDTS != null)
+            versionInputModel.setGUID(textGUIDTS.getText());
+    }
+
     public void onPressedChangeVersion() {
-        // need to be set from text fields by darpan
         if (validateInput(tabbedPaneUI.getTitleAt(tabbedPaneUI.getSelectedIndex()))) {
-            VersionInputModel versionInputModel = new VersionInputModel("C:\\Idera\\ERStudioFiles", "20.2.2.0", "20.2.3.0", "1234", UpgradeTypeEnum.MAJOR_OR_MINOR_UPGRADE);
+            VersionInputModel versionInputModel = new VersionInputModel("", "", "", "", UpgradeTypeEnum.PATCH_UPGRADE);
```
Comment (vaibhavjain3) [inline src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java:451]: Why this has been made empty string?

**Review by vaibhavjain3 — CHANGES_REQUESTED** [review]

File: src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java (hunk 1 above)
Comment (Akshit-Accolite) [reply src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java:190]: done

File: src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java (hunk 2 above)
Comment (Akshit-Accolite) [reply src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java:451]: Hey Vaibhav the function in the next line copyComponentsFromGUI is used to update these values

**Pull request merged**